Update Ingress support for kubernetes version 1.18

GooseCoder opened the pull request:
When I try the library with Kubernetes version 1.18 I got an error, digging up it seems that the API has changed from "extensions" to "networking.k8s.io" on this component. This should fix it. But also note that this could break apps running on older versions. Maybe it should be better to add a new branch that supports version 1.18 but I leave it to your discretion. 
Regards, and nice job with this library I used it for various Laravel apps interacting with k8s clusters.

maclof commented:
The current plan with this project is to always support the latest version of Kubernetes.
While i'd like to support every version, I don't think it's very feasible at the moment.
Please fix the unit tests then I'll be happy to merge.

maclof commented:
Please also update the README.md file under the "Supported API features" heading.

GooseCoder commented:
Hi @maclof, I've updated the PR, unit tests are passing now, and also I've updated the README doc with this API change. Please let me know if anything else is needed.
Best regards,
Gustavo
